charts/appdynamics-cluster-agent/20.7.0/questions.yaml

```diff
@@ -0,0 +1,188 @@
+labels:
+  io.rancher.certified: partner
+categories:
+    - monitoring
+    - appdynamics
+questions:
```

Comment:
Can you remove the `description` of all questions where it's the same or too similar to the `label`? The description should be used to extend on details when the `label` is not self explanatory, otherwise it's okay to leave it out to not clutter the UI.

Reply:
Got it. THX!